### ISSUE TITLE
Test missing from the distribution

### ISSUE THREAD

**Issue by AMDmi3** [pjknkda/flake8-datetimez#1]
```shell
% wget -qO- https://files.pythonhosted.org/packages/source/f/flake8-datetimez/flake8-datetimez-19.4.5.0.tar.gz | tar -tzf-
flake8-datetimez-19.4.5.0/
flake8-datetimez-19.4.5.0/PKG-INFO
flake8-datetimez-19.4.5.0/README.md
flake8-datetimez-19.4.5.0/flake8_datetimez.egg-info/
flake8-datetimez-19.4.5.0/flake8_datetimez.egg-info/PKG-INFO
flake8-datetimez-19.4.5.0/flake8_datetimez.egg-info/SOURCES.txt
flake8-datetimez-19.4.5.0/flake8_datetimez.egg-info/dependency_links.txt
flake8-datetimez-19.4.5.0/flake8_datetimez.egg-info/entry_points.txt
flake8-datetimez-19.4.5.0/flake8_datetimez.egg-info/requires.txt
flake8-datetimez-19.4.5.0/flake8_datetimez.egg-info/top_level.txt
flake8-datetimez-19.4.5.0/flake8_datetimez.py
flake8-datetimez-19.4.5.0/setup.cfg
flake8-datetimez-19.4.5.0/setup.py
```

`test_datetimez.py` is missing, `python setup.py test` fails (`ModuleNotFoundError: No module named 'test_datetimez'`). Probably should be added to `packages` or `package_data`, not sure.

**Comment by pjknkda** [comment]
Have been fixed in 19.5.4.0 release. Thank you for reporting!

**Comment by AMDmi3** [comment]
Now it works, thanks!
